go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/giantswarm/chart-operator
 
-go 1.13
+go 1.14
```

Comment:
architect-orb is already 0.8.8+ which means it would built by go 1.14 when we build images, but for safety reason I would like to update it as 1.14.